sympy/concrete/summations.py

```diff
@@ -989,22 +991,21 @@ def eval_sum_symbolic(f, limits):
         r = gosper_sum(f, (i, a, b))
 
         # to find Sum at points where denominator approaches to zero
-        from sympy import together, fraction, Mul, Add, ordered
-        if isinstance(r, (Mul,Add)) and not(any(x in r.atoms(Symbol) for x in limits)):
-            var = ordered(r.atoms(Symbol))
-            _, denom = fraction(together(r))
-            if any(x in denom.atoms(Symbol) for x in var):
+        if isinstance(r, (Mul,Add)):
+            from sympy import ordered
+            var = r.free_symbols
+            var = [x for x in var if x not in limits]
+            den = denom(together(r))
+            if any(x in den.free_symbols for x in var):
```

Comment:
Your logic here allows you to enter the cond enumerating process if *any* symbols are in common but then you put conditions on *all* symbols. (If den didn't have a certain symbol then it was just skipped so what you had worked.) You also have an impossible condition `m == False and m == True` so you add every condition.

I suspect that you mean

```
cond = []
for v in ordered(var & den.free_symbols):
    for s in solve(den, v):
        m = Eq(v, s)
        if not m:  # I don't think it can ever be True, can it?
            continue
        cond.append(m)
```